### DISCUSSION TITLE
Notation PR review criteria

### DISCUSSION THREAD

**Discussion by gokarnm** [notaryproject/notation#263] [General]
1. A code reviewer MUST be either a member of *[notation-code-reviewers](https://github.com/orgs/notaryproject/teams/notation-code-reviewers)* or *[notation-sub-group-maintainers](https://github.com/orgs/notaryproject/teams/notation-sub-group-maintainers)*.
1. Every PR MUST require 2 code review approvals before a PR merge.
1. Both reviewers MAY be from the same maintainer organization if a PR has been open for more than a week.
1. If review feedback from reviewer of a different maintainer organization is present, it MUST be addressed before PR can be merged. In this case at least one code review approval from the other maintainer organization MUST be required before PR can be merged.
    1. Minor feedback MAY be addressed by creating tracking issues against it. 
1. A member of *notation-sub-group-maintainers* SHOULD only merge a PR after it has two approvals from either *notation-code-reviewers* or *notation-sub-group-maintainers*.

**Comment by dtzar** [comment]
I would also suggest for non-critical PRs where there are two different code-reviewer approvals that we utilize the calls to merge prior to the week to help speed things up.
